includes/class-ur-form-handler.php

```diff
@@ -774,6 +774,104 @@ public function get_form_fields( $form_id, $args = array() ) {
 		}
 		return $fields_dict;
 	}
+
+
+	/**
+	 * Create new form.
+	 *
+	 * @since  2.2.4
+	 * @param  string $title    Form title.
+	 * @param  string $template Form template.
+	 * @param  array  $args     Form Arguments.
+	 * @param  array  $data     Additional data.
+	 * @return int|bool Form ID on successful creation else false.
+	 */
+	public function create( $title = '', $template = 'blank', $args = array(), $data = array() ) {
+
+		if ( empty( $title ) ) {
+			return false;
+		}
+
+		$args         = apply_filters( 'user_registration_create_form_args', $args, $data );
+
+		// Prevent content filters from corrupting JSON in post_content.
+		$has_kses = ( false !== has_filter( 'content_save_pre', 'wp_filter_post_kses' ) );
+		if ( $has_kses ) {
+			kses_remove_filters();
+		}
+		$has_targeted_link_rel_filters = ( false !== has_filter( 'content_save_pre', 'wp_targeted_link_rel' ) );
+		if ( $has_targeted_link_rel_filters ) {
+			wp_remove_targeted_link_rel_filters();
+		}
+
+		$templates = ur_get_json_file_contents( 'assets/extensions-json/templates/all_templates.json' );
+
+		$form_data = array();
+
+		if ( ! empty( $templates ) ) {
+			foreach ( $templates->templates as $template_data ) {
+				if ( $template_data->slug === $template && 'blank' !== $template_data->slug ) {
+					error_log( print_r( json_decode( base64_decode( $template_data->settings ), true ), true ) );
```

Comment:
You missed to remove the log function here.

Reply:
Removed !!!